ixofoundation/ixo-webclient
# chore: update project hero styling

## Scope
https://nonaredteam.atlassian.net/browse/IXO-350

## Work Done
Updated the layout of the hero component
Updated the styling
Added the ability to display a country flag

## Steps to Test
Check that the styling looks like the designs
To see a flag displayed you will need to change the project data slightly by adding `project.projectLocation = 'ZA'`

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/79502394-d7c43600-802f-11ea-9749-34731f0daa8b.png)
